src/mobile/src/libs/device.js

```diff
@@ -2,15 +2,16 @@ import { Platform, PermissionsAndroid } from 'react-native';
 import DeviceInfo from 'react-native-device-info';
 import i18next from 'shared-modules/libs/i18next';
 
-const device = DeviceInfo.getModel();
 const deviceId = DeviceInfo.getDeviceId();
 
 export const locale = DeviceInfo.getDeviceLocale();
 export const timezone = DeviceInfo.getTimezone();
 
 export const isAndroid = Platform.OS === 'android';
 export const isIOS = Platform.OS === 'ios';
-export const isIPhoneX = device.includes('iPhone X') || deviceId.includes('iPhone11') || deviceId.includes('iPhone12');
+export const isIPhoneX =
+    (deviceId === 'iPhone10,3' || deviceId === 'iPhone10,6' || parseInt(deviceId.substring(6, 8)) >= 11) &&
```

Comment:
```suggestion
export const isIPhoneX =
    isIOS &&
    (deviceId === 'iPhone10,3' || deviceId === 'iPhone10,6' || parseInt(deviceId.substring(6, 8)) >= 11) &&
```

Without `isIOS` there's an edge case where this could be true on an Android. `deviceId` uses `android.os.Build.BOARD` which could contain a value such as `GT-I9195`, which would make `isIPhoneX` true